### DISCUSSION TITLE
Astroid is fantastic but I can’t see how spread the word!

### DISCUSSION THREAD

**Discussion by paurray** [templaza/astroid-framework#375] [Show and tell]
Hello Joomla & Astroid Folks

I am a video editor & on the side Joomla developer.
I made the first of what I hope will be more courses about Astroid.
They are a lot of fun to make & thanks to the Astroid community for everything.

The Channel I set up is here:

https://www.youtube.com/@Astroid-Joomla-Seamlessly

There is a First Playlist here:

First Bare Bones Website with Astroid & Joomla
https://www.youtube.com/playlist?list=PLOd7DxRu2yaJvVJKPEGZAOGoWpWOyNL1b

With a couple of other videos thrown in for good measure. All live & Online. Now.

The problem is that I have no idea where to spread the word in the Joomla world.

Can you give me a nudge in the right direction or help spread the word? 

Thanks

Paul

**Comment by paurray** [comment]
Any tips would be appreciated. I am thinking Joomla Groups on Facebook? Maybe on a Joomla forum, Reddit or what ever. I just do not know enough about where people hang out in the Joomla world as in where I could spread the word about Astroid specifically the tutorials I made about Astroid.

**Comment by paurray** [comment]
I guess a good place to start:

Joomla Developers
https://www.facebook.com/groups/246796345390331/

**Comment by vtheod** [comment]
You can also check here:
https://www.facebook.com/groups/joomlaextensions [https://www.facebook.com/groups/joomlaextensions]

On 30/8/2023 10:37:20, Paul David Murray ***@***.***> wrote:
I guess a good place to start:
Joomla Developers
https://www.facebook.com/groups/246796345390331/ [https://www.facebook.com/groups/246796345390331/]
—
Reply to this email directly, view it on GitHub [https://github.com/templaza/astroid-framework/discussions/375#discussioncomment-6860975], or unsubscribe [https://github.com/notifications/unsubscribe-auth/AD3ES4DFRFNJ4R2UK52BEFTXX3US7ANCNFSM6AAAAAA3ZWNS24].
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
[275fc239-6f2c-4878-b4e2-802c1dd753d7]

**Comment by paurray** [comment]
@vtheod Thanks for this. I have posted there and they seem to be mostly friendly :-)

**Comment by vtheod** [comment]
You're very welcome buddy!
On 27/8/2023 22:11:36, Paul David Murray ***@***.***> wrote:
Any tips would be appreciated. I am thinking Joomla Groups on Facebook? Maybe on a Joomla forum, Reddit or what ever. I just do not know enough about where people hang out in the Joomla world as in where I could spread the word about Astroid specifically the tutorials I made about Astroid.
—
Reply to this email directly, view it on GitHub [https://github.com/templaza/astroid-framework/discussions/375#discussioncomment-6836164], or unsubscribe [https://github.com/notifications/unsubscribe-auth/AD3ES4BY4YBM3FQYKMASWQDXXOLWPANCNFSM6AAAAAA3ZWNS24].
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
[10906b0e-dc2a-4647-8f73-0f2ff62d27ad]

**Comment by paurray** [comment]
Also trying my luck here:
https://www.reddit.com/r/joomla/comments/16ajp6v/your_1st_bare_bones_website_with_astroid_joomla/
Any body got other ideas?
